test-services/kong/Dockerfile

```diff
@@ -1,8 +1,18 @@
-FROM kong:0.14-alpine
-
+ARG KONG_VERSION=1.0.0-centos
+FROM kong:${KONG_VERSION}
+RUN yum install -y epel-release
+RUN yum install -y git sudo
 WORKDIR /usr/local/share/lua/5.1/kong
 RUN sed -i '38ilua_shared_dict kong_signalfx_aggregation 10m;' templates/nginx_kong.lua
 RUN sed -i '38ilua_shared_dict kong_signalfx_locks 100k;' templates/nginx_kong.lua
 RUN sed -i '29i\ \ "signalfx",' constants.lua
-RUN luarocks install --server=http://luarocks.org/manifests/signalfx kong-plugin-signalfx
```

Comment:
Are you unable to install from luarocks?

Reply:
> Are you unable to install from luarocks?

I get this error:
```Step 9/15 : RUN luarocks install --server=http://luarocks.org/manifests/signalfx kong-plugin-signalfx
 ---> Running in 90bd64aadcba
Warning: The directory '/root/.cache/luarocks' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing /usr/local/bin/luarocks with sudo, you may want sudo's -H flag.
Warning: Failed searching manifest: Failed extracting manifest file
Warning: Failed searching manifest: Failed extracting manifest file
Installing https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/kong-plugin-signalfx-0.0.2-1.src.rock

Error: Failed unpacking rock file: /tmp/luarocks_luarocks-rock-kong-plugin-signalfx-0.0.2-1-7YK2Wu/kong-plugin-signalfx-0.0.2-1.src.rock
The command '/bin/sh -c luarocks install --server=http://luarocks.org/manifests/signalfx kong-plugin-signalfx' returned a non-zero code: 1```

Reply:
for posterity: looked into this and unzip is a dependency not available in the kong centos images for whatever reason.

Also `--server=http://luarocks.org/manifests/signalfx` is unnecessary now.  I think luarocks has added it to the root manifest.